src/IssueViz.Security/Hotspots/HotspotsList/HotspotsControl.xaml

```diff
@@ -96,7 +116,7 @@
                         </DataTemplate>
                     </DataGridTemplateColumn.CellTemplate>
                 </DataGridTemplateColumn>-->
-                <DataGridTemplateColumn CanUserSort="True" SortMemberPath="Hotspot.Issue.Rule.Priority" Header="Priority" Width="90"  MinWidth="64">
```

Comment:
Unrelated bugfix, but it was too small to create a separate PR IMO

Reply:
What was the effect of the bug - an empty column?

Reply:
Is it sortable now?

Reply:
Should be. I will test before merging

Reply:
Hmmm still doesn't work

Reply:
I will create a tickets as I also just realised that even if it worked, it could have sorted it based on alphabetical order, which means it would be High, Low, Medium - and that's not what we want

Reply:
[#4584](https://github.com/SonarSource/sonarlint-visualstudio/issues/4584)